src/world/GameWorldBase.h

```diff
@@ -62,6 +61,8 @@ class GameWorldBase : public World
     GameWorldBase(const std::vector<GamePlayer>& players, const GlobalGameSettings& gameSettings, EventManager& em);
     ~GameWorldBase() override;
 
+
+    boost::interprocess::unique_ptr<LuaInterfaceGame, Deleter<LuaInterfaceGame> > lua;
```

Comment:
Instead of making it public create a function `SetLua(LuaInterfaceGame*)` and then you could even make it private as we have `SetLua`, `HasLua` and `GetLua`